frontend/src/components/Header/Header.jsx

```diff
@@ -62,9 +62,15 @@ const Header = () => {
                         Accéder à mon espace
                       </ToolItem>
                     )}
-                    <ToolItem asLink={<div data-test-id="logout-button" onClick={deleteToken} />}>
-                      Déconnexion
-                    </ToolItem>
+                    <div
+                      data-test-id="logout-button"
+                      className="clickable"
+                      onClick={deleteToken}
+                    >
+                      <ToolItem asLink={<div />}>
+                        Déconnexion
+                      </ToolItem>
+                    </div>
```

Comment:
I though the burger close fix was an update from react-dsfr.. Couldn't integrate it in the library directly?

Reply:
yes, but we still need a div to say that when you click you logout...